test.js

```diff
@@ -106,6 +106,21 @@ test('custom template', () => {
 	});
 });
 
+test('custom async template', () => {
+	return compiler(
+		{},
+		getConfig({
+			context: { title: 'Pizza', htmlAttributes: { lang: 'it' } },
```

Comment:
Maybe drop `htmlAttributes` since it's not used in this test.

Reply:
Ah yeah, I just copy-pasted it...